[DEVX-1634] Add saucectl documentation for excludedTestFiles (#1223)
saucelabs/sauce-docs: docs/web-apps/automated-testing/testcafe/yaml.md

```
---

### `excludedTestFiles`
<p><small>| OPTIONAL | ARRAY/REGEX |</small></p>

> Excludes test files to skip the tests. Files are matched by shell pattern, e.g. the explicit name, file glob, or the location of the test files.

Reviewer: ```suggestion
Excludes test files to skip the tests. Files are matched by shell pattern, such as the explicit name, file glob, or location of the test files.
```
